sdk/eidolon_ai_sdk/builtins/logic_units/api_logic_unit.py

```diff
@@ -0,0 +1,70 @@
+from typing import List, Optional, Any
+
+import jsonref
+from pydantic import BaseModel, Field
+
+from eidolon_ai_client.util.logger import logger
+from eidolon_ai_sdk.apu.call_context import CallContext
+from eidolon_ai_sdk.apu.logic_unit import LogicUnit
+from eidolon_ai_sdk.builtins.logic_units.api_helper import get_content, build_call
+from eidolon_ai_sdk.builtins.logic_units.openapi_helper import Operation, build_actions, Action
+from eidolon_ai_sdk.system.fn_handler import FnHandler
+from eidolon_ai_sdk.system.reference_model import Specable
+from eidolon_ai_sdk.util.schema_to_model import schema_to_model
+
+
+class ApiLogicUnitSpec(BaseModel):
+    title: str
+    root_call_url: str
+    open_api_location: str
+    operations_to_expose: List[Operation]  # map of tool name to openapi path
+    key_env_var: str
+    key_query_param: Optional[str]
```

Comment:
what is key_env_var / key_query_param?

descriptions here would help so that it gets documented well on our website.

Reply:
Yep -- forgot to do that